### PR TITLE
Feat / kup-typography

### PR DESCRIPTION
**Pull request by Leonardo-Signorelli**

### CODE REVIEW

File: packages/ketchup/src/f-components/f-typography/f-typography.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    props: FTypographyProps,
+    children: VNode[]
+) => {
+    if (!props.type) {
+        props.type = FTypographyType.HEADING1;
+    }
+    if (!props.value) {
+        return;
+    }
+    if (!props.toolbar) {
```
Comment (pasere-smeup) [inline packages/ketchup/src/f-components/f-typography/f-typography.tsx:20]: props.toolbar si boolean; set to false is different than not set, but here will go into the if if undefined OR false

File: packages/ketchup/src/typography.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
```
Comment (pasere-smeup) [inline packages/ketchup/src/typography.html:1]: this example doesn't work!

**Review by pasere-smeup — CHANGES_REQUESTED** [review]
Added some comments into the code, please check.

Showcase example is missing.

File: packages/ketchup/src/components/kup-tab-bar/kup-tab-bar-declarations.ts [hunk 3]
```diff
@@ -24,8 +24,3 @@ export interface KupTabBarEventPayload extends KupEventPayload {
     node: KupTabBarNode;
     index: number;
 }
-
-export interface KupTabbarItemClickEventPayload extends KupEventPayload {
-    value?: string;
```
[the remark this answers is not in the document]
Comment (Leonardo-Signorelli) [reply packages/ketchup/src/components/kup-tab-bar/kup-tab-bar-declarations.ts:29]: I have already removed this Event as Toolbar events are now isolated inside the toolbar declarations and re-called inside components

File: packages/ketchup/src/components/kup-typography/kup-typography.tsx [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+     */
+    private toolbarList: KupDynamicPositionElement;
+    #dropDownActionCardAnchor: HTMLElement = null;
+
+    /*-------------------------------------------------*/
+    /*                   E v e n t s                   */
+    /*-------------------------------------------------*/
+
+    @Event({
+        eventName: 'kup-typography-icon-click',
```
[the remark this answers is not in the document]
Comment (Leonardo-Signorelli) [reply packages/ketchup/src/components/kup-typography/kup-typography.tsx:102]: Fixed

**Review by pasere-smeup — APPROVED** [review]
ok

**Pull request merged**